src/components/Courses/components/CourseCard/tests/courseCard.test.js

```diff
@@ -0,0 +1,47 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { mockedAuthorsList, mockedCoursesList } from '../../../../../constants';
+import CourseCard from '../CourseCard';
+
+const mockedState = {
+	user: {
+		isAuth: true,
+		name: 'Test Name',
+		token: 'test',
+	},
+	courses: [],
+	authors: mockedAuthorsList,
+};
+
+const mockedStore = {
+	getState: () => mockedState,
+	subscribe: jest.fn(),
+	dispatch: jest.fn(),
+};
+
+const course = mockedCoursesList[0];
+
+test('CourseCard should display all course details in correct format', () => {
```

Comment:
usually we start the tests with the word should, no need to mention component because you write tests for specific component. this is relevant for all your tests description

Reply:
Changed!